app/src/lib/menu-update.ts

```diff
@@ -127,13 +127,18 @@ function getMenuState(state: IAppState): Map<MenuIDs, IMenuItemState> {
     'show-history',
     'show-repository-list',
     'show-branches-list',
+    'new-repository',
+    'add-local-repository',
+    'clone-repository',
```

Comment:
@iAmWillShepherd These three doesn't look like they're repository scoped, i.e they don't require a selected repository. We should move this to explicit enable/disable calls further down.